### DISCUSSION TITLE
& was unexpected at this time.

### DISCUSSION THREAD

**Discussion by sjdsdhzhdohazdoazhd** [firemodels/fds#13120] [Q&A] [answered]
<img width="860" alt="question fds" src="https://github.com/firemodels/fds/assets/157407229/5cdcfc43-f1db-47cd-b0d5-2bce86c69ab2">
Hi, community.
I am new to FDS and Smokeview and am trying to run the examples. But I just met this problem: No matter what I type, FDS always shows & was unexpected at this time (as shown in the image attached). can anybody help me with that pls?
Thanks

**Comment by mcgratta** [comment]
I do not understand how you are trying to run the job. What command are you invoking?

**Reply by sjdsdhzhdohazdoazhd** [reply]
<img width="604" alt="image" src="https://github.com/firemodels/fds/assets/157407229/0f2a53ee-744c-4fda-a1fe-cd1c81cc6667">

This one: &HEAD CHID='extinction_1', TITLE='Test_Case_For_Extinction_Models' / I think this command enables to name the output file (this is a command from the extinction example from FDS) But no matter what command I type, it always show : & was unexpected at this time.

**Comment by drjfloyd** [comment]
please read the User's Manual. It has detailed instructions on how to run FDS on different operating systems.

**Reply by sjdsdhzhdohazdoazhd** [reply]
it is windows system, i think it just click and run no?

**Reply by mcgratta** [reply]
No. FDS doesn't work that way. Read the User's Guide.

**Reply by sjdsdhzhdohazdoazhd** [reply]
could u pls give me some information pls? i read chpiters 2 and 3. they just introduce how to use multiple processors and check the Computer Hardware. This is the guide for Single Computer Running MS Windows: "Open up the special FDS command prompt, CMDfds, which should appear on your desktop when you install FDS. By opening this special command prompt, a script is run automatically ensuring that the FDS commands and libraries are all consistent." I don't think the user's guide include the solution of the problem...

**Reply by mcgratta** [reply]
![image](https://github.com/firemodels/fds/assets/11333911/9a1250b1-bde6-4962-a7f1-17b5be2805ec)

**Reply by sjdsdhzhdohazdoazhd** [reply]
yes, i read it. but i think "Open up the special FDS command prompt, CMDfds, which should appear on your desktop when you
install FDS. By opening this special command prompt, a script is run automatically ensuring that the FDS
commands and libraries are all consistent." this means exactly that I just need to click to open it. Then I put input file in the same directory and try the prompt
<img width="679" alt="image" src="https://github.com/firemodels/fds/assets/157407229/392b71e0-f0c9-4dda-b02b-7c12e405105c">
it still shows & was unexpected at this time.
I think I will try another PC or ask my advisor if it still doesn't work

**Reply by mcgratta** [reply]
You can run this case using a single MPI process (p) and a single OpenMP thread (o):
```
fds_local -p 1 -o 1 extinction_1.fds
```

**Reply by sjdsdhzhdohazdoazhd** [reply]
Thank u for ur advise. it successfully completed the example program. But it still shows the same thing...
<img width="442" alt="image" src="https://github.com/firemodels/fds/assets/157407229/8233080f-7ba6-4d65-8d01-3b53c1af24fc">
it is too late in France. i will try some other ways to solve this problem tomorrow
Thank u very much for your info

**Reply by drjfloyd** [reply, marked as answer]
You cannot type the individual lines of input at a prompt. You have to put them all in a file.

**Reply by sjdsdhzhdohazdoazhd** [reply]
yes, you are correct, i tried the command fds example.fds and smokeview example
it works
Thank you for ur help
